conf/application.conf

```diff
@@ -68,7 +69,7 @@ application.langs="en"
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-    enabled = true
+    enabled = false
```

Comment:
There's a few places in `InternalAuthTokenInitialiserISpec` where this is also being set to false. As that's now redundant would you mind removing those?